criu/uffd.c

```diff
@@ -269,7 +269,11 @@ int uffd_open(int flags, unsigned long *features)
 
 	uffd = syscall(SYS_userfaultfd, flags);
 	if (uffd == -1) {
-		pr_info("Lazy pages are not available: %s\n", strerror(errno));
+		if (flags == 0) {
```

Comment:
I wouldn't rely on `flags` to select the logging level. Consider adding an explicit parameter to `uffd_open` and then use this parameter.
Another possibility is to drop the `pr_error()` if `syscal(SYS_userfaultfd)` fails and update the error message in `setup_uffd()`:
```
diff --git a/criu/uffd.c b/criu/uffd.c
index 33b34ba25..b0baff08a 100644
--- a/criu/uffd.c
+++ b/criu/uffd.c
@@ -315,7 +315,7 @@ int setup_uffd(int pid, struct task_restore_args *task_args)
         */
        task_args->uffd = uffd_open(O_CLOEXEC | O_NONBLOCK, &features);
        if (task_args->uffd < 0) {
-               pr_perror("Unable to open an userfaultfd descriptor");
+               pr_perror("Lazy pages are not available");
                return -1;
        }